core/GStreamer/Pipeline.vala

```diff
@@ -47,6 +49,9 @@ public class Noise.Pipeline : GLib.Object {
 
         pipe = new Gst.Pipeline("pipeline");
         playbin = Gst.ElementFactory.make ("playbin", "play");
+        
+        fake = Gst.ElementFactory.make ("fakesink", "fake");
+        playbin.set ("video-sink", fake );
```

Comment:
`playbin["video-sink"] = fake` would be better (or even `playbin.video_sink = fake`, but not sure it will work).